igvc_navigation/launch/refactored_mapper.launch

```diff
@@ -1,13 +1,13 @@
 <launch>
     <!-- Move Base Flex -->
-    <node pkg="mbf_costmap_nav" type="mbf_costmap_nav" respawn="false" name="move_base_flex" output="screen">libsimple_layer
+    <node pkg="mbf_costmap_nav" type="mbf_costmap_nav" respawn="false" name="move_base_flex" output="screen" >
         <param name="tf_timeout" value="1.5"/>
         <param name="planner_max_retries" value="3"/>
         <rosparam file="$(find igvc_navigation)/config/planners.yaml" command="load" />
         <rosparam file="$(find igvc_navigation)/config/controllers.yaml" command="load" />
         <rosparam file="$(find igvc_navigation)/config/recovery_behaviors.yaml" command="load" />
-        <rosparam file="$(find igvc_navigation)/config/empty_global_costmap_params.yaml" command="load" />
```

Comment:
We should remove these config files as well since they were just temporary solutions until the changes were done.

Reply:
Yupp